components/Favorites/utils.js

```diff
@@ -0,0 +1,200 @@
+export const colors = {
+  RED: 'Красное',
+  WHITE: 'Белое',
+  ROSE: 'Розовое',
+}
+export const sugar = {
+  SWEET: 'Сладкое',
+  DRY: 'Сухое',
+  MEDIUM_DRY: 'Полусухое',
+  MEDIUM: 'Полусладкое',
+}
+export const stores = {
+  'winelab.ru': 'WineLab',
+  'perekrestok.ru': 'Перекрёсток',
+  'https://spb.winestyle.ru': 'WineStyle',
+  'https://www.lenta.com': 'Лента',
+  'amwine.com': 'Ароматный мир',
+  'https://simplewine.ru': 'SimpleWine',
+}
+export const sortingButtons = [
+  {
+    value: 'recommendations',
+    textLabel: 'Рекомендованные',
+  },
+  {
+    value: 'priceAsc',
+    textLabel: 'Возрастанию цен',
+  },
+  {
+    value: 'priceDesc',
+    textLabel: 'Убыванию цен',
+  },
+  {
+    value: 'popularity',
+    textLabel: 'Популярности',
+  },
+]
+export const sortDesc = wines => {
```

Comment:
Can we combine both desc and asc sorts onto one sorting function that will receive the sorting method? 
like string 'asc' | 'desc' or boolean 0 | 1
decide which one suits u the best :)